src/pages/(study)/admin/components/AdminTabs.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * 관리자 페이지 탭 컴포넌트
+ */
+
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { ADMIN_TABS } from '../constants';
+import type { AdminTabType } from '../types';
+
+interface AdminTabsProps {
+  activeTab: AdminTabType;
+}
+
+export const AdminTabs: React.FC<AdminTabsProps> = ({ activeTab }) => {
+  // 탭 ID를 상대 URL로 매핑
+  const getTabUrl = (tabId: string) => {
+    switch (tabId) {
+      case 'members':
+        return 'members';
+      case 'applicants':
+        return 'applicants';
+      case 'study-info':
+        return 'study-info';
+      default:
+        return 'members';
+    }
+  };
+
+  return (
+    <div className='flex space-x-1 mb-6'>
+      {ADMIN_TABS.map((tab) => (
+        <Link
+          key={tab.id}
+          to={getTabUrl(tab.id)}
```

Comment:
```suggestion
          to={tab.id ?? 'members'}
```
getTabUrl 메소드 없이 그냥 이렇게만 해도 되지 않나 싶어서요~